provision/acc_provision/flavors.yaml

```diff
@@ -153,6 +153,145 @@ flavors:
     config:
       kube_config:
         no_priority_class: True
+  l2-tenant-chained-multus-openshift-4.12-baremetal:
+    desc: Red Hat OpenShift Container Platform 4.12 on Baremetal
+    default_version: 6.0
+    config:
+      net_config:
+        kubeapi_vlan_mode: untagged
+      kube_config:
+        use_acicni_priority_class: True
+        use_external_service_ip_allocator: False
+        host_agent_openshift_resource: True
+        allow_pods_external_access: True
+        use_privileged_containers: True
+        use_openshift_security_context_constraints: False
+        use_openshift_run_level: True
+        use_cnideploy_initcontainer: True
```

Comment:
use_cnideploy_initcontainer should be False as we are not the primary CNI.

Reply:
Resolved.